source/_integrations/matter.markdown

```diff
@@ -29,8 +31,6 @@ Matter is [the new standard for home automation](https://en.wikipedia.org/wiki/M
 
 Matter devices are available using either WiFi based communication or [Thread](/integrations/thread/), both are supported by Home Assistant. Bluetooth is used for adding new devices to your Matter network.
 
-Home Assistant only supports control of Matter devices. Home Assistant is not a bridge itself and it cannot turn devices within Home Assistant into Matter compatible devices.
```

Comment:
Why did you remove this ?

Reply:
Yes, I realized I shouldn't remove here, sorry